openmp/runtime/src/dllexports

```diff
@@ -1268,6 +1268,11 @@ kmp_set_disp_num_buffers                    890
     __kmpc_atomic_val_8_cas_cpt            2158
     %endif
 
+    # No longer need to put ordinal numbers
```

Comment:
This will not cause issue on Windows anymore?

Reply:
Right, you can add exported symbols just by name now. Older symbols keep their ordinal number.